### DISCUSSION TITLE
[Feature Request] Add ethers.js v6 support

### DISCUSSION THREAD

**Discussion by feloot** [zkSync-Community-Hub/zksync-developers#31] [Feature requests]
### Network

zkSync Era

### Provide a brief description of the feature you would like to see implemented.

Add ethers.js v6 support, ideally without the need for any third party library. If following standards, ethers.js should be able to process zkSync Era txs without any issue. Please see : https://github.com/ethers-io/ethers.js/issues/3934

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

All projects that are up to date with ethers.js v6 and are simply left out of zkSync because of the lack of support.

**Comment by dutterbutter** [comment]
🙌 Thank you for your feature request! We really appreciate it! 😊 We apologize on the delay of ethers v6 support. We have made progress here and blockers have been removed. We'll keep you in the loop with any updates. 

Keep these feature requests coming, as they help us prioritize features and make our product better!

**Comment by keytwojohn** [comment]
Integrating ethers.js v6 support directly into zkSync Era would greatly benefit developers and projects utilizing the latest version of ethers.js. By ensuring compatibility with industry standards, such as ethers.js v6, zkSync Era can seamlessly process transactions without requiring additional third-party libraries. This enhancement not only enhances usability but also expands the accessibility of zkSync Era to a broader range of projects, ultimately fostering greater adoption and innovation within the ecosystem.

**Comment by kimballthomas** [comment]
Adding ethers.js v6 support to zkSync Era would greatly benefit users, developers, and the protocol itself. It enables projects utilizing ethers.js v6 to seamlessly integrate with zkSync, expanding the ecosystem and facilitating adoption. This enhancement aligns zkSync with current standards, enhancing interoperability and user experience.

**Comment by tovanpanet** [comment]
Implementing ethers.js v6 support benefits both developers and users by enabling seamless integration of zkSync transactions with projects using the latest version of ethers.js, enhancing interoperability and expanding the ecosystem.

**Comment by amandakob** [comment]
Adding ethers.js v6 support in zkSync Era benefits developers and users, enabling seamless integration with projects utilizing the latest version of ethers.js. This enhances interoperability and expands zkSync's ecosystem to a broader range of applications.

**Comment by shannonzimmons** [comment]
The user is requesting the addition of ethers.js v6 support to zkSync Era, ideally without the need for any third-party library. This feature would benefit all projects that are up to date with ethers.js v6 and are currently left out of zkSync because of the lack of support. Following standards, ethers.js should be able to process zkSync Era transactions without any issue. Further discussion and implementation may be needed to determine the feasibility and impact of this feature on the zkSync Era network.

**Comment by dutterbutter** [comment]
This is completed.
